src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/MetaController.php

```diff
@@ -283,4 +301,46 @@ public function generateRobotsFileAction()
 
         return $this->redirectToRoute('admin_metas_index');
     }
+
+    /**
+     * Saves the user preference of closing the showcase card
+     *
+     * @AdminSecurity("is_granted(['create', 'update'], request.get('_legacy_controller'))")
+     * @DemoRestricted(redirectRoute="admin_metas_index")
+     *
+     * @return JsonResponse
+     */
+    public function closeShowcaseCardAction(Request $request)
```

Comment:
wouldnt it be better to have `ShowcaseCardController:closeShowcaseCardAction`, since this action is excatly the same as it would be for any other card, expect it could accept additonal card name parameter in request?

Reply:
I guess it should be an ApiController, right